vllm/engine/metrics.py

```diff
@@ -166,7 +166,7 @@ def __init__(self, labelnames: List[str], vllm_config: VllmConfig):
         #   Latency
         request_latency_buckets = [
             0.3, 0.5, 0.8, 1.0, 1.5, 2.0, 2.5, 5.0, 10.0, 15.0, 20.0, 30.0,
-            40.0, 50.0, 60.0
+            40.0, 50.0, 60.0, 90.0, 120.0, 180.0, 240.0, 360.0, 480.0
```

Comment:
I'm not aware of anyone today running 16m or 32m request to a language model today.  However, an 8B model with 128k context could in theory generate about 20 tokens per request per second (at a TPOT of 50ms at ~15 req/s on an H100).  If there is only one very long output request running and a lot of prefill heavy requests you could imagine  completing that in 6400s or 106m.

So there is at least some data to suggest that these bucket sizes may even be too small today for today's models.

Reply:
If metric cardinality could be a problem, should we consider doubling or quadruling the bucket size past 1 minute? Perhaps 4 minutes, 16 minutes, and 64 minutes?

Reply:
Thanks @smarterclayton @RenaultAI 
It has been changed :-)